domains/caleba.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "CISAFounder",
+        "email": "calebinternet@outlook.com"
+    },
+    "record": {
+        "URL": "cisainternet.com"
```

Comment:
URL needs protocol (also site redirects to IONOS saying that there isnt a website connected yet)